CONTRIBUTING.md

```diff
@@ -39,6 +39,17 @@ currently it is not possible commit changes to Git LFS if you are not
 a team member, so you need to ask someone from the team for help if
 your PR includes changes to assets.
 
+Unfortunately, issues are often created quickly with little detail
+and context. Please do not hesistate to ask questions regarding the
+issue for clarification and details.
```

Comment:
I wanted this paragraph to be after the 2 other added ones, because those are related to finishing a PR, so they logically go with the "Once your changes are complete..." paragraph.
And this issue information thing, I think maybe this should be put under the "Github Issues" heading.